documentation/how-to-guides/deploy_magma_access_gateway.md

```diff
@@ -0,0 +1,63 @@
+# How-to: Deploy Magma Access Gateway using snap
+
+The goal of this document is to guide the operator through the process of deploying 
+Magma Access Gateway using [magma-access-gateway](https://snapcraft.io/magma-access-gateway) snap.
+
+> **WARNING:** **magma-access-gateway** snap will affect your computer's networking configuration.
+> Make sure it is installed on designated hardware (personal computers are strongly discouraged).
+
+### System requirements
+
+**Hardware (baremetal strongly recommended)**
+
+- Processor: x86-64 dual-core processor (around 2GHz clock speed or faster)
+- Memory: 4GB RAM
+- Storage: 32GB or greater SSD
+
+**Networking**
+
+- At least two ethernet interfaces (SGi and S1)
+- Internet connectivity from SGi interface
+
+**Operating System**
+
+- Ubuntu 20.04 LTS
+  ([Ubuntu installation guide](https://help.ubuntu.com/lts/installation-guide/amd64/index.html))
+
+## 1. Install
+
+> **NOTES:**<br>
+> 1. To see the list of currently supported configuration options, execute:
+> ```bash
+> magma-access-gateway.install --help
+> ```
+> 2. Unless specified otherwise, all actions in this guide are executed using `root` account.<br>
+> 3. Installation progress can be monitored at all times using `journalctl`.
+
+Connect to the machine where you want to install Magma Access Gateway and execute the following command as root:
```

Comment:
```suggestion
Connect to the machine where you want to install Magma Access Gateway and execute the following command:
```

Reply:
No need for the "as root" since you placed the notes before. I added the "root" because I had proposed initially to place the notes after. But I think it's fine like it is (if we remove the "as root")